doc/_static/css/custom.css

```diff
@@ -0,0 +1,74 @@
+:root {
+    --pst-color-primary: 47,47,47;
+    --pst-color-link: 56,123,178;
+    --pst-color-link-hover: 254,203,56;
```

Comment:
These should be adapted to the HoloViews colors if they haven't already.

Reply:
Ok done, the links are now in the same color as the font of the holoviews logo.